### PR TITLE
Handle text commands

### LINKED ISSUE #604: Implement temporary processing for text commands in channels
This temporary module should parse commands from chat messages and invoke the corresponding methods provided by the module created for #605. This will be the predecessor to #607.

### PR DESCRIPTION
**Pull request by amgassert**
Closes #604

### CODE REVIEW

**Comment by GreenMobius** [comment]
this isn't going into the final channels release, but allows testing of channels implementations independently of the UI.

File: packages/frontend/app/services/messages/command-handler.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    new TextCommand(
+      'change-user-context-remove-member',
+      ['ucr'],
+      ['user'],
+      this.changeUserContextRemoveMember
+    ),
+    new TextCommand('view-user-context', ['ucv'], ['user'], this.viewUserContext),
+  ];
+
+  //TODO: get the author and channel of the command
```
Comment (CelticMajora) [inline packages/frontend/app/services/messages/command-handler.ts:60]: Should this TODO be handled in this PR or a future one?

File: packages/frontend/app/services/messages/command-handler.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      commandString += ' ';
+      commandString += '[' + parameter + ']';
+    });
+    commandString += '\n';
+    return commandString;
+  }
+}
+
+export default class CommandHandler extends Service {
+  @service user!: CurrentUserService; //author = user.record()
```
Comment (CelticMajora) [inline packages/frontend/app/services/messages/command-handler.ts:39]: The only thing is just remove the comments.

File: packages/frontend/app/services/messages/command-handler.ts [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    }
+    textCommand.execute(args);
+  }
+
+  printState(): void {
+    //TODO
+  }
+
+  addMember(_args: string[]): void {
+    //TODO
```
Comment (NullVoxPopuli) [inline packages/frontend/app/services/messages/command-handler.ts:102]: are these blocked by anything?

**Comment by amgassert** [comment]
Closing in accordance with new plans

**Pull request closed without merging**